Add a note about settings requestAutoCert to False explicitly when providing certificates to minio (#1184)
minio/operator: docs/tls.md

kubectl create secret generic tls-ssl-minio --from-file=path/to/private.key --from-file=path/to/public.crt
```

> Once created, set the name of Secret (here it is `tls-ssl-minio`) under `spec.externalCertSecret` field. Then create the Tenant. MinIO Operator will use this Secret to fetch key and certificate and mount it to relevant locations inside the Tenant pods. Make sure you explicitly set `requestAutoCert` field to `false` to disable automatic generation.

Reviewer: This is not necessary however the wording should be made optional here, I think this document update is not necessary or should be written differently.

> Once created, set the name of Secret (here it is `tls-ssl-minio`) under `spec.externalCertSecret` field. Then create the Tenant. MinIO Operator will use this Secret to fetch key and certificate and mount it to relevant locations inside the Tenant pods. Make sure you explicitly set `requestAutoCert` field to `false` to disable automatic generation.

Reviewer: ```suggestion
Once created, set the name of Secret (here it is `tls-ssl-minio`) under `spec.externalCertSecret` field. Then create the Tenant. MinIO Operator will use this Secret to fetch the key and certificate and mount it to relevant locations inside the Tenant pods. 

NOTE: Additionally to disable automatic cert generation make sure the `spec.requestAutoCert` field to `false`.
```

      type: kubernetes.io/tls
```

> Make sure you explicitly set `requestAutoCert` field to `false` to disable automatic generation.

Reviewer: ```suggestion
Additionally to disable automatic cert generation make sure the `spec.requestAutoCert` field to `false`.
```
